src/pkg/services/m365/api/users.go

```diff
@@ -301,6 +318,62 @@ func (c Users) GetMailInbox(
 	return inbox, nil
 }
 
+func (c Users) GetMailboxInfo(
```

Comment:
This is a copy of the mailbox code from `GetInfo`, right?  The other GetInfo funcs are marked as deprecated, but not the one in this file.  Does this allow us to get rid of that func as well?

Reply:
Yep, correct